scripts/us_census/geojsons_low_res/requirements.txt

```diff
@@ -0,0 +1,7 @@
+absl-py==0.9.0
```

Comment:
As discussed verbally, let's update the global `requirements.txt` instead of adding this file.

Reply:
Done.